kit/media/src/UIVideo/UIVideo.native.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import RNVideo from 'react-native-video';
```

Comment:
Since we do not need to use `react-native-video` for web, you can exclude it from the final bundling by setting an alias for it in **webpack.js**, i.e.:
```
'react-native-video': 'react', // Hack in order not to load
```